src/muxy.ts

```diff
@@ -197,6 +199,14 @@ export class Muxy implements MuxyInterface {
   public messenger: Messenger;
 
   /**
+   * Internal {@link Transactions}.
+   *
+   * @ignore
+   * @type {Transactions}
+   */
+   public transaction: PurchaseClient;
```

Comment:
This field shouldn't be called `transaction` anymore. `purchaseClient` works for me.